themes/classic/templates/_partials/head.tpl

```diff
@@ -43,10 +43,22 @@
     <link rel="canonical" href="{$page.canonical}">
   {/if}
   {block name='head_hreflang'}
-      {foreach from=$urls.alternative_langs item=pageUrl key=code}
-            <link rel="alternate" href="{$pageUrl}" hreflang="{$code}">
-      {/foreach}
+    {foreach from=$urls.alternative_langs item=pageUrl key=code}
+      <link rel="alternate" href="{$pageUrl}" hreflang="{$code}">
+    {/foreach}
   {/block}
+  
+  {include file="_partials/microdata/head-jsonld.tpl"}
+  
+  {block name='head_microdata_special'}{/block}
+  
+  {include file="_partials/pagination-seo.tpl"}
```

Comment:
this should definitely be in block, what about third party modules like `Advanced Search` or `Amazzing filters`? I think it would not be compatible

Reply:
`  {block name='head_pagination_seo'}
    {include file="_partials/pagination-seo.tpl"}
  {/block}`